jenkins/e2e-image/e2e-runner.sh

```diff
@@ -525,3 +537,10 @@ if [[ "${JENKINS_SOAK_MODE:-}" == "y" && ${E2E_UP:-} == "true" ]]; then
   echo ${KUBERNETES_RELEASE} | gsutil cp - "${JENKINS_SOAK_PREFIX}/release.txt"
   echo ${KUBERNETES_RELEASE_URL} | gsutil cp - "${JENKINS_SOAK_PREFIX}/release-url.txt"
 fi
+
+# If we are only deploying federated clusters without the federation control plane,
+# then the kubeconfig for these clusters will be required while deploying the federation
+# control plane. So we persist the kubeconfig in GCS for later use.
+if [[ "${FEDERATION_UP:-}" == "true" && -z "${FEDERATION_CLUSTERS:-}" ]]; then
+  gsutil cp "${JENKINS_FEDERATION_PREFIX}/kube-config" "${HOME}/.kube/config"
+fi
```

Comment:
These 3 lines are almost the same as the block which starts with line no [477](https://github.com/kubernetes/test-infra/pull/1982/files#diff-6276ace2db72f1df8d04fb42c6989b69R477). am i missing something here?